src/main/java/com/endava/soa_5/array_and_string/MatchedTimes.java

```diff
@@ -0,0 +1,39 @@
+package com.endava.soa_5.array_and_string;
+
+public class MatchedTimes {
+    MatchedTimes() {
+    }
+
+    // this method multiply array values and return final result
+    public static int multiplicationOfArray(int[] array) {
+        int multiplication = 1;
+        int index = 0;
+        while (index < array.length) {
+            multiplication = multiplication * array[index];
```

Comment:
You can do it shorter:  multiplication *=  array[index];

Reply:
Done, but this was out of scope!